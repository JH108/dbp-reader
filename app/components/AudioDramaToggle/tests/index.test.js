import React from 'react';
// import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import { AudioDramaToggle } from '..';

const props = {
	audioType: 'audio_drama',
	dispatch: jest.fn(),
};

describe('<AudioDramaToggle />', () => {
	it('Expect to match snapshot', () => {
		const tree = renderer.create(<AudioDramaToggle {...props} />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
