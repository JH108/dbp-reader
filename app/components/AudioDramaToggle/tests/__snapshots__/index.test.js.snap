// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AudioDramaToggle /> Expect to match snapshot 1`] = `
<div
  className="audio-drama-toggle-container"
>
  <button
    className="audio-drama-toggle-button active"
    id="drama-button"
    onClick={[Function]}
    type="button"
  >
    Drama
  </button>
  <button
    className="audio-drama-toggle-button"
    id="non-drama-button"
    onClick={[Function]}
    type="button"
  >
    Non-Drama
  </button>
</div>
`;
